src/cascadia/TerminalControl/ControlCore.h

```diff
@@ -191,6 +191,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         TYPED_EVENT(RaiseNotice,               IInspectable, Control::NoticeEventArgs);
         TYPED_EVENT(TransparencyChanged,       IInspectable, Control::TransparencyChangedEventArgs);
         TYPED_EVENT(ReceivedOutput,            IInspectable, IInspectable);
+        TYPED_EVENT(FoundMatch,            IInspectable, Control::FoundResultsArgs);

```

Comment:
manual spacing is off here

Reply:
![i-am-ashamed](https://user-images.githubusercontent.com/18356694/151865564-b7e60fc6-d8dd-4422-9e33-0008545e2e58.gif)